Wikipedia/Code/WMFColumnarCollectionViewLayout.h

```diff
@@ -0,0 +1,24 @@
+#import <UIKit/UIKit.h>
+
+@class WMFCVLMetrics;
+
+/*!
+ @class        WMFColumnarCollectionViewLayout
+ @abstract     A WMFColumnarCollectionViewLayout organizes a collection view into columns grouped by section - all items from the same section will be in the same column.
+ @discussion   ...
+ */
+@interface WMFColumnarCollectionViewLayout : UICollectionViewLayout
```

Comment:
I think this is fine… I just missed that you actually did the annotations just didn't need assume non null